src/features/events/components/EventOverviewCard/EventParticipantsCard.tsx

```diff
@@ -0,0 +1,221 @@
+                cursor="pointer"
+                onClick={(event) =>
+                  setAnchorEl(anchorEl ? null : event.currentTarget)
+                }
+              />
+            </Box>
+            <Popper anchorEl={anchorEl} open={!!anchorEl}>
+              <ClickAwayListener
+                onClickAway={() => {
+                  setAnchorEl(null);
+                  if (
+                    newReqParticipants != null &&
+                    newReqParticipants != reqParticipants
+                  ) {
+                    model.setReqParticipants(newReqParticipants);
+                  }
+                }}
+              >
+                <Paper elevation={3} variant="elevation">
+                  <Box mt={1} p={2}>
+                    <TextField
+                      helperText={messages.eventParticipantsCard.reqParticipantsHelperText()}
+                      label={messages.eventParticipantsCard.reqParticipantsLabel()}
+                      onChange={(ev) => {
+                        const val = ev.target.value;
+
+                        if (val == '') {
+                          setNewReqParticipants(null);
+                          return;
+                        }
+
+                        const intVal = parseInt(val);
+                        if (!isNaN(intVal) && intVal.toString() == val) {
+                          setNewReqParticipants(intVal);
+                        }
+                      }}
+                      onKeyDown={(ev) => {
+                        if (ev.key === 'Enter') {
+                          setAnchorEl(null);
+                          if (newReqParticipants != null) {
+                            model.setReqParticipants(newReqParticipants);
+                          }
+                        } else if (ev.key === 'Escape') {
+                          setAnchorEl(null);
+                          setNewReqParticipants(reqParticipants);
+                        }
+                      }}
+                      value={
+                        newReqParticipants === null ? '' : newReqParticipants
+                      }
+                      variant="outlined"
+                    />
+                  </Box>
+                </Paper>
+              </ClickAwayListener>
+            </Popper>
+          </Box>
+        }
+      >
+        <Divider />
+        <Box
+          display="flex"
+          flexDirection="column"
+          justifyContent="space-between"
+        >
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginY={1}
+          >
+            <Typography
+              color={theme.palette.text.secondary}
+              component="h6"
+              variant="subtitle1"
+            >
+              {messages.eventParticipantsCard.pending()}
+            </Typography>
+            <Typography>
+              {Math.max(reqParticipants - availParticipants, 0)}
```

Comment:
Clever! :D

Reply:
I think the correct way to calculate this is actually slightly more complicated, and requires additional data. I have solved a similar situation in #1199 (pending review), so maybe leave this like this for now and we can fix it once both PRs are merged.